src/assets/js/splash-screen.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+  var app = document.getElementById("app");
+  var splashContainer = document.getElementById("splash-container");
+  if (app && splashContainer) {
+
+    var removeSplashContainer = function () {
+      splashContainer.parentElement.removeChild(splashContainer);
+    };
+
+    // remove the splash screen on any click
+    splashContainer.onclick = function () {
+      clearTimeout(checkerTimeout);
+      removeSplashContainer();
+    };
+
+    var checkerTimeout;
```

Comment:
NBD, but because `checkerTimeout` is used on line 12, I was looking for `checkerTimeout` somewhere near line 4 or 5.